src/promptflow/setup.py

```diff
@@ -46,7 +46,7 @@
     "strictyaml>=1.5.0,<2.0.0",  # used to identify exact location of validation error
     "waitress>=2.1.2,<3.0.0",  # used to serve local service
     "opencensus-ext-azure<2.0.0",  # configure opencensus to send telemetry to azure monitor
-    "ruamel.yaml>=0.17.35,<0.18.0",  # used to generate connection templates with preserved comments
+    "ruamel.yaml>=0.17.35,<0.19.0",  # used to generate connection templates with preserved comments
```

Comment:
Please ensure CI is using new version: 
![image](https://github.com/microsoft/promptflow/assets/47586720/fac35c03-9c69-41ed-bc6d-525903174296)

Reply:
let me double check